app/build.gradle

```diff
@@ -201,6 +207,7 @@ android {
 dependencies {
     implementation project(':solanaj')
     implementation project(':ui-kit')
+    implementation project(':ui-kit-sample')
```

Comment:
it seems to me that app have all dependencies
can you check size of app with this impl and without ?

i do not like it if we will have more then we need in our app like size (classes / resources/ etc)

Reply:
33 MB vs 32 MB